op-node/flags/flags.go

```diff
@@ -182,8 +182,12 @@ var (
 	L1CacheSize = &cli.UintFlag{
 		Name: "l1.cache-size",
 		Usage: "Cache size for blocks, receipts and transactions. " +
-			"It's optional and a sane default of 3/2 the sequencing window size is used if this field is set to 0.",
+			"If this field is set to 0, a default of 3/2 the sequencing window size is used. " +
```

Comment:
Should this say "1/4 the sequencing window"? As far as I can tell the [standard sequencing window is 3600 L1 blocks](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/op.toml#L12). So 1/4 * 3600 = 900, which is the default value this pr introduces

Reply:
It's indeed wrong, it should be "2/3", not "3/2". Note that the default is the value of `Value = 900`. Only if this is set to `0` is `2/3 seq window` chosen.